docs/_source/getting_started/installation/configurations/user_management.md

```diff
@@ -346,18 +346,39 @@ http.get("/api/workspaces").json()
 ````
 
 
-### Migrate users with docker-compose
-
-Make sure you create the yaml file above in the same folder as your `docker-compose.yaml`. You can download the `docker-compose` from this [URL](https://raw.githubusercontent.com/argilla-io/argilla/main/docker-compose.yaml):
+### Migrate users with Docker Compose
+
+Make sure you create the YAML file above in the same folder as your `docker-compose.yaml`. You can download the `docker-compose.yaml` file from this [URL](https://raw.githubusercontent.com/argilla-io/argilla/main/docker-compose.yaml):
+
+Then open the provided `docker-compose.yaml` file and modify your Argilla instance as follows:
+
+```diff
+  argilla:
+    image: argilla/argilla-server:latest
+    restart: unless-stopped
+    ports:
+      - "6900:6900"
+    environment:
+      ARGILLA_HOME_PATH: /var/lib/argilla
+      ARGILLA_ELASTICSEARCH: http://elasticsearch:9200
++     ARGILLA_LOCAL_AUTH_USERS_DB_FILE: /var/lib/argilla-migrate/users.yaml
+    networks:
+      - argilla
+    volumes:
+      - argilladata:/var/lib/argilla
++     - ${PWD}/users.yaml:/var/lib/argilla-migrate/users.yaml
+```
 
-Then open the provided ``docker-compose.yaml`` and configure your Argilla instance as follows:
+After that change you can start the containers with:
 
-{{ dockercomposeuseryaml }}
+```bash
+docker compose up
+```
 
-Then, you can run the migrate tasks as follows:
+And after running the containers you can run the task to migrate the users as follows:
 
 ```bash
-docker-compose exec argilla python -m argilla.tasks.users.migrate
+docker compose exec argilla python -m argilla.tasks.users.migrate
```

Comment:
This won't work for old docker versions, but the `-` version is yes.

Reply:
Let's change them then.

Reply:
Done.